notebooks/239-freevc/239-freevc.ipynb

```diff
@@ -0,0 +1,733 @@
+{
```

Comment:
<p>for people who out of context of this model, it is not clear what you mean under helpers?</p><p> </p><p>Can it be rephrased?</p><p>Just: For preparing input for inference, we should define helper functions based on <code>speaker_encoder.voice_encoder.SpeakerEncoder class methods</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>